AdminWebsite/AdminWebsite/Controllers/UserIdentityController.cs

```diff
@@ -54,20 +54,21 @@ public async Task<ActionResult<UserProfileResponse>> GetUserProfile()
         }
         
         /// <summary>
-        /// Get the Justice User list from the JusticeUser table
+        /// Get list of Justice User filtered by passed term. If term is null then not filter applied.
```

Comment:
could we tweak this description slightly:

`Get list of Justice User filtered by term. If term is null then no filter applied.`

Reply:
done